### PR TITLE
fix: fixed error in the file

### LINKED ISSUE #1066: Update method creation syntax to ES6 `IconSet.js`
### 🆕🐥☝ First Timers Only.

This issue is reserved for people who never contributed to Open Source before. We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

[About First Timers Only](http://www.firsttimersonly.com/).

### 🤔 What you need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below.

  Once claimed we add you as contributor to this repository.

- [ ] 👌 **Accept our invitation** to this repository. Once accepted, assign yourself to this issue

- [ ] 📝 **Update** the file [\src/iconsets/IconSet.js](https://github.com/publiclab/Leaflet.DistortableImage/blob/main/src%2Ficonsets%2FIconSet.js) in the `Leaflet.DistortableImage` repository (press the little pen Icon) and edit the line as shown below.

```diff
@@ -6,12 +6,12 @@ L.IconSet = L.Class.extend({
 
   _symbols: '',
 
-  render: function() {
+  render() {
     this.addSymbols(this._symbols);
     return this._svg;
   },
 
-  addSymbols: function(symbols) {
+  addSymbols(symbols) {
     this._svg += symbols;
   },
 });
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/)

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions

Leave a comment below!

This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).

### PR DESCRIPTION
**Pull request by Khadeeejah**
Fixes #1066 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [ ] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by Khadeeejah** [comment]
hi @TildaDares  kindly review my pr , thank you

File: src/iconsets/IconSet.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+ * we don't use it directly */
 L.IconSet = L.Class.extend({
-
   _svg: '<svg xmlns="http://www.w3.org/2000/svg">',
 
   _symbols: '',
 
-  render: function() {
-    this.addSymbols(this._symbols);
-    return this._svg;
```
Comment (TildaDares) [inline src/iconsets/IconSet.js:11]: Hi @Khadeeejah, please undo this change.

File: src/iconsets/IconSet.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
-* we don't use it directly */
+ * we don't use it directly */
 L.IconSet = L.Class.extend({
-
   _svg: '<svg xmlns="http://www.w3.org/2000/svg">',
 
   _symbols: '',
 
-  render: function() {
-    this.addSymbols(this._symbols);
```
Comment (TildaDares) [inline src/iconsets/IconSet.js:10]: Please undo this change!

File: src/iconsets/IconSet.js [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
-    return this._svg;
+  render() {
+    this.addSymbols(this._symbols)
+    return this._svg
   },
 
-  addSymbols: function(symbols) {
-    this._svg += symbols;
```
Comment (TildaDares) [inline src/iconsets/IconSet.js:15]: Please undo this change

File: src/iconsets/IconSet.js [hunk 4]
```diff
@@ -1,17 +1,16 @@
...
+  addSymbols(symbols) {
+    this._svg += symbols
   },
-});
```
Comment (TildaDares) [inline src/iconsets/IconSet.js:17]: Please undo this change!

File: src/iconsets/IconSet.js [hunk 5]
```diff
@@ -1,17 +1,16 @@
 /* this is the baseclass other IconSets inherit from,
-* we don't use it directly */
+ * we don't use it directly */
 L.IconSet = L.Class.extend({
-
```
Comment (TildaDares) [inline src/iconsets/IconSet.js:4]: Please undo this change!

File: src/iconsets/IconSet.js [hunk 6]
```diff
@@ -1,17 +1,16 @@
 /* this is the baseclass other IconSets inherit from,
-* we don't use it directly */
```
Comment (TildaDares) [inline src/iconsets/IconSet.js:2]: Please undo this change!

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @Khadeeejah, it seems you have a formatter/linter configured in your IDE. Please turn it off so it doesn’t add unwanted changes. Thank you!

**Comment by Khadeeejah** [comment]
Hi @TildaDares  sorry about the linter issue, i have fixed the issues

**Review by TildaDares — APPROVED** [review]
Amazing work @Khadeeejah. Thank you!!

**Comment by TildaDares** [comment]
Congratulations on merging your PR @Khadeeejah 🎉

**Comment by jywarren** [comment]
Great job!!

**Comment by Khadeeejah** [comment]
Hi can i start creating new issues for first timers or look for more issues to work on as i am willing to work on more @TildaDares

**Comment by TildaDares** [comment]
Hi @Khadeeejah, the choice is up to you but it would be very helpful if you could help create issues for other first timers. Thank you!

**Pull request merged**